### PR TITLE
TW-620-implement make offer for alien

### PR DESCRIPTION
**Pull request by Boltman92**

### CODE REVIEW

File: src/screens/collection/components/collectible-item.tsx [hunk 1]
```diff
@@ -131,14 +130,14 @@ export const CollectibleItem: FC<Props> = ({ item, collectionContract, setWidth
   const handleList = () => openUrl(navigateToObjktForBuy);
 
   const buttonText = () => {
```
Comment (lendihop) [inline src/screens/collection/components/collectible-item.tsx:132]: variable instead of fn and useMemo

**Review by lendihop — APPROVED** [review]

File: src/screens/collection/components/collectible-item.tsx [hunk 2]
```diff
@@ -173,46 +172,45 @@ export const CollectibleItem: FC<Props> = ({ item, collectionContract, setWidth
             </View>
           </View>
         </View>
-        {isHolder && (
-          <View style={styles.buttonContainer}>
+
+        <View style={styles.buttonContainer}>
```
[the remark this answers is not in the document]
Comment (Boltman92) [reply src/screens/collection/components/collectible-item.tsx:176]: I did not change `root` class. What is the question?

[the remark this answers is not in the document]
Comment (svjatoslavtt) [reply src/screens/collection/components/collectible-item.tsx:176]: `useCollectionStyles` use in 2 files

**Review by svjatoslavtt — APPROVED** [review]

**Review by vladvn1905 — APPROVED** [review]
ios done build: 1.15_722

**Review by IrynaKhyzhynska — APPROVED** [review]
android done 1.15_722

**Pull request merged**